r/ruamel-ordereddict/ruamel-ordereddict_ubi_8.5.sh

```diff
@@ -0,0 +1,59 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: ruamel-ordereddict
+# Version	: 0.4.15
+# Source repo	: http://hg.code.sf.net/p/ruamel-ordereddict/code
+# Tested on	: UBI 8.5
+# Language      : Python
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Atharv Phadnis <Atharv.Phadnis@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=ruamel-ordereddict
+PACKAGE_VERSION=${1:-0.4.15}
+PACKAGE_URL=http://hg.code.sf.net/p/ruamel-ordereddict/code
+
+set -e
+
+yum -y update && yum install -y python2 python2-devel gcc gcc-c++ make redhat-rpm-config
+python2 -m pip install Mercurial
```

Comment:
Is python2 hard requirement? Can we use python3?

Reply:
@Siddhesh-Ghadi Yes, the package doesn't support python3, we can refer to comments on its issues like: https://sourceforge.net/p/ruamel-ordereddict/tickets/8/#ae3d

Reply:
Okay. Thanks.